src/rgw/rgw_data_sync.cc

```diff
@@ -747,12 +764,16 @@ class RGWListBucketIndexesCR : public RGWCoroutine {
   int i;
 
   bool failed;
+  string list_marker;
+  int max_entries;
+  bool truncated;
+  read_metadata_list result;
```

Comment:
do we need member variables for `list<string> keys`, `string list_marker`, and `bool truncated` when we can already access them inside `result`?

Reply:
Right, no need